canonical-web-and-design/jaas-dashboard
# Add UI for edit config if the charm has no configuration options

## Done

Added UI for charms which have no configuration options

## QA

- Open demo.
- View `charmed-kubernetes` model.
- Click `easyrsa` and then `configuration`.
- See the UI in the screenshot below.
- Close the panel
- Click `containerd` and then `configuration`.
- See the edit configuration UI.

## Details

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1580

## Screenshots

![Screen Shot 2021-01-19 at 3 36 40 PM](https://user-images.githubusercontent.com/532033/105096057-29409c00-5a6c-11eb-90b0-21fcb05d1772.png)
